apps/web/lib/features/team/user-team-card/user-info.tsx

```diff
@@ -75,9 +75,9 @@ export function UserInfo({ className, memberInfo, publicTeam = false }: Props) {
 									? 'pause'
 									: !member?.employee?.isActive && !publicTeam
 									? 'suspended'
-									: member?.employee?.isOnline &&
-									  member?.timerStatus !== 'running'
-									? 'online'
+									: member?.employee?.isOnline
+									? //  && member?.timerStatus !== 'running'
```

Comment:
@badalkhatri0924 I don't get it? Why commenting it out will make it more accurate?

Reply:
@evereq when timer is running we are getting isOnline true.

Ideally timeStatus should return all the cases. Like Online, Online_and_tracking time etc. That I think we will do later on when we have real time.